frontend/src/layout/navigation/navigationLogic.ts

```diff
@@ -27,6 +28,7 @@ export const navigationLogic = kea<navigationLogicType<UserType, SystemStatus, W
         setPinnedDashboardsVisible: (visible: boolean) => ({ visible }),
         setInviteMembersModalOpen: (isOpen: boolean) => ({ isOpen }),
         setHotkeyNavigationEngaged: (hotkeyNavigationEngaged: boolean) => ({ hotkeyNavigationEngaged }),
+        setFilteredEnvironment: (environment: string) => ({ environment }),
```

Comment:
type of `environment` variable should be the enum/const options, not just any string.